package.json

```diff
@@ -94,6 +97,7 @@
     "start": "webpack-dev-server --devtool=cheap-module-source-map --inline"
   },
   "dependencies": {
-    "jsdoc": "^3.4.0"
+    "jsdoc": "^3.4.0",
+    "react-redux": "^4.4.1"
```

Comment:
I realize I don't know whether there's a significant reason to use `dependencies` over `devDependencies`.